src/nvim/buffer.c

```diff
@@ -2143,8 +2118,8 @@ buf_T *buflist_findnr(int nr)
     nr = curwin->w_alt_fnum;
   }
 
-  if (pmap_has(handle_T)(buf_map, (handle_T)nr)) {
-    return pmap_get(handle_T)(buf_map, (handle_T)nr);
+  if (handle_exists_buffer((handle_T)nr)) {
```

Comment:
This is redundant, `pmap_get`  will return NULL on nonexisting key per convention.